pkg/providers/instance/instance.go

```diff
@@ -51,7 +51,7 @@ import (
 
 var (
 	// MaxInstanceTypes defines the number of instance type options to pass to CreateFleet
-	MaxInstanceTypes                 = 60
+	MaxInstanceTypes                 = 100
```

Comment:
Can we just drop this value if we aren't truncating over here anymore?

Reply:
Yeah, I dropped this. But it is also used in a test though to make sure LT would have the 100 cheapest instancetype as part of CreateFleet call since that is still relevant. We just happen to do this upstream now. So, I have just hard coded it for the test.